.github/workflows/check-pr.yml

```
@@ -24,17 +24,14 @@ jobs:
       - name: Setup Flutter with FVM
         uses: ./.github/actions/setup-fvm
       
-      - name: Get dependencies
-        working-directory: ${{ env.WORKING_DIR }}
-        run: flutter pub get
+      - name: Run melos analyze
+        run: melos run analyze
       
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)